content/blog/breaks/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Breaks
+date: "2019-04-16T22:12:03.284Z"
+description: Replenish your energy.
+---
+
+> A break at work is a period of time during a shift in which an employee is allowed to take time off from his/her job. (Wikipedia).
+
+### Guilt
+
+Workers often feel guilty if they stand up and take a break. It's especially true for people in tech and the newcomers like interns or junior programmers. Hustling hard is the new norm, and I don't have anything against hard work, but play it smart to avoid getting [burned out](https://devhealth.io/burnout/).
+
+### Benefits
+
+Taking scheduled breaks during work is essential. It improves focus, keeps you from getting bored and helps you figure out a problem that you are stuck on.
+
+When you are frustrated and stressing out over a hard task, taking a break and putting the work in subconscious mind may help solve it.
+
+Walking, talking to coworkers or playing a video game can help you relax, and the insights will come more often when in a relaxed state.
+
+![Break](./break.jpg)
+
+> "Walk and talk." One type of meetings that became popular recently is the walking meeting. Instead of sitting in a conference room, take a more active and personal approach and do a round or two around the block while talking and discussing. It's great if you need to address a creative problem or brainstorm about a problem solution.
+
+### Lunch break
+
+Great way to relax, socialize and de-stress during the workday is to take a lunch break. Almost all employers allow a 20-30 min break for lunch.
+
+In some Asian countries, it's common to take an after lunch nap. It's true for both factory and office workers. You can also observe people sleeping on public transportation. The napping culture and the benefits from restored energy after the short nap can be seen in some high tech companies as they provide sleeping pods or beds for employees.
```

Comment:
_It's true for_
if you mean this like 'to velja za ...' then it would be better to use sth like 'this is a practice among' or 'this occurs among'